jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/impl/AttributeCollector.java

```diff
@@ -402,21 +402,21 @@ private void addRawPropertyValue(ObjectNode node, String propertyName, Object va
 
     private void addRawArrayItem(ArrayNode node, Object value) {
         // need to specifically add simple/primitive values by type
-        if (value instanceof String) {
+        if (value instanceof String string) {
             // explicit inclusion as string results in wrapping quote symbols
-            node.add((String) value);
-        } else if (value instanceof BigDecimal) {
-            node.add((BigDecimal) value);
-        } else if (value instanceof BigInteger) {
-            node.add((BigInteger) value);
-        } else if (value instanceof Boolean) {
-            node.add((Boolean) value);
-        } else if (value instanceof Double) {
-            node.add((Double) value);
-        } else if (value instanceof Float) {
-            node.add((Float) value);
-        } else if (value instanceof Integer) {
-            node.add((Integer) value);
+            node.add(string);
+        } else if (value instanceof BigDecimal decimal) {
+            node.add(decimal);
+        } else if (value instanceof BigInteger integer) {
+            node.add(integer);
+        } else if (value instanceof Boolean boolean1) {
+            node.add(boolean1);
+        } else if (value instanceof Double double1) {
+            node.add(double1);
+        } else if (value instanceof Float float1) {
+            node.add(float1);
```

Comment:
**suggestion**: Let's please avoid ambiguous naming wherever possible, including not assigning numbered variable name, even for such short-lived ones.
```suggestion
        } else if (value instanceof Boolean booleanValue) {
            node.add(booleanValue);
        } else if (value instanceof Double doubleValue) {
            node.add(doubleValue);
        } else if (value instanceof Float floatValue) {
            node.add(floatValue);
```